modules/pico-sdk/1.6.0-rc1/presubmit.yml

```diff
@@ -0,0 +1,14 @@
+bcr_test_module:
+  module_path: bazel/example
+  matrix:
+    platform: ["debian10", "ubuntu2004", "macos", "macos_arm64", "windows"]
+  tasks:
+    run_test_module_build:
+      name: "Build test"
+      platform: ${{ platform }}
+      bazel: "7.x"
+      build_flags:
+        - "--platforms=@pico-sdk//bazel/platform:rp2040"
```

Comment:
looks like you also need `--@rules_cc//cc/toolchains:experimental_enable_rule_based_toolchains` here